parts/intro.adoc

```diff
@@ -5,7 +6,7 @@ Contexts and Dependency Injection
 
 [.topic]
 == What is CDI?
-image::cdi_icon.png[role="pull-right", width="335"]
+image::/images/cdi_icon.png[role="pull-right", width="335"]
```

Comment:
Asciidoctor finds the image path thanks to :imagesdir: attributes in slides.adoc